tests/serverpod_test_server/lib/src/endpoints/test_tools.dart

```diff
@@ -180,6 +182,30 @@ class TestToolsEndpoint extends Endpoint {
   ) async {
     return simpleDatas;
   }
+
+  Future<void> logMessageWithSession(Session session) async {
+    session.log('test session log in endpoint');
+  }
+
+  static Completer<void> willCloseListenerCalled = Completer();
```

Comment:
Suggestion: Change this to a nullable that is assigned in each test.

Then we can use a local instance in the test.

Reply:
c3d9f33